src/main/java/com/faforever/moderatorclient/ui/domain/MatchmakerQueueFX.java

```diff
@@ -0,0 +1,38 @@
+package com.faforever.moderatorclient.ui.domain;
+
+import javafx.beans.property.SimpleStringProperty;
+import javafx.beans.property.StringProperty;
+
+public class MatchmakerQueueFX extends AbstractEntityFX {
+    private StringProperty technicalName;
+    private StringProperty nameKey;
+
+    public MatchmakerQueueFX() {
+        technicalName = new SimpleStringProperty();
```

Comment:
initialize on variable declaration and remove constructor

Reply:
Use lombok @Data and remove unnecessary getter and setter

Reply:
Same as MapPoolFx